client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -121,8 +132,16 @@ export default React.createClass( {
 	},
 
 	getInstallButton() {
-		if ( this.hasInstallButton() ) {
-			return <PluginInstallButton { ...this.props } />;
+		if ( this.props.selectedSite && this.props.selectedSite.jetpack ) {
+			if ( this.hasOrgInstallButton() ) {
+				return <PluginInstallButton { ...this.props } />;
```

Comment:
do we have to blindly pass in all props or would we be fine explicitly passing down the ones that matter? would make the code much easier to use to track data flows